README.md

```diff
@@ -413,6 +413,39 @@ Replies with a JSON array containing the following fields:
 - `Hubname` contains the Azure IoT Hub hubname string.
 - `Port` contains the Azure IoT Hub MQTT port number.
 
+## `api/v1/cc/{serial}` X509 Certificate Copy Request: **GET**
+
+Requests the x509 certificate copy based on the supplied certificate serial number.
+
+If you need information about the serial number refer 
+[here](#apiv1csserial-certificate-status-request-get)
+
+### Request by default `application/json`
+
+You can send the x509 certificate copy request via:
+
+```bash
+$ curl -v --cacert certs/CA.crt  \
+          --cert certs/BOOTSTRAP.crt \
+          --key certs/BOOTSTRAP.key  \
+          https://MBP2021.local:1443/api/v1/cc/1664276589086394391
+```
```

Comment:
We'll also need this to work with CBOR, though the default should be JSON when no content type is specified.